src/main/resources/project/project.xml

```diff
@@ -9694,6 +9745,21 @@
                 <type>textarea</type>
             </formalParameter>
 
+            <formalParameter>
+                <formalParameterName>syncCells</formalParameterName>
+                <defaultValue/>
+                <description>If checked, syncCells will be triggered after deployment.</description>
+                <required>0</required>
+                <type>textarea</type>
```

Comment:
Seems that it has no effect since we have form.xml, but I think it still should be changed to checkbox